rcon/utils.py

```diff
@@ -163,33 +172,39 @@ def get_map_side(map_):
 LONG_HUMAN_MAP_NAMES = {
     "carentan_offensive_ger": "Carentan Offensive (GER)",
     "carentan_offensive_us": "Carentan Offensive (US)",
+    "carentan_warfare_night": "Carentan (Night)",
     "carentan_warfare": "Carentan",
     "driel_offensive_ger": "Driel Offensive (GER)",
     "driel_offensive_us": "Driel Offensive (UK)",
+    "driel_warfare_night": "Driel (Night)",
     "driel_warfare": "Driel",
     "elalamein_offensive_CW": "El Alamein Offensive (UK)",
     "elalamein_offensive_ger": "El Alamein Offensive (GER)",
+    "elalamein_warfare_night": "El Alamein (Night)",
     "elalamein_warfare": "El Alamein",
     "foy_offensive_ger": "Foy Offensive (GER)",
     "foy_offensive_us": "Foy Offensive (US)",
     "foy_warfare_night": "Foy (Night)",
     "foy_warfare": "Foy",
     "hill400_offensive_ger": "Hill 400 Offensive (GER)",
     "hill400_offensive_US": "Hill 400 Offensive (US)",
+    "hill400_warfare_night": "Hill 400 (Night)",
     "hill400_warfare": "Hill 400",
     "hurtgenforest_offensive_ger": "Hürtgen Forest Offensive (GER)",
     "hurtgenforest_offensive_US": "Hürtgen Forest Offensive (US)",
     "hurtgenforest_warfare_V2_night": "Hürtgen Forest (Night)",
     "hurtgenforest_warfare_V2": "Hürtgen Forest",
     "kharkov_offensive_ger": "Kharkov Offensive (GER)",
     "kharkov_offensive_rus": "Kharkov Offensive (RUS)",
+    "kharkov_warfare_night": "Kharkov (Night)",
     "kharkov_warfare": "Kharkov",
     "kursk_offensive_ger": "Kursk Offensive (GER)",
     "kursk_offensive_rus": "Kursk Offensive (RUS)",
     "kursk_warfare_night": "Kursk (Night)",
     "kursk_warfare": "Kursk",
     "omahabeach_offensive_ger": "Omaha Beach Offensive (GER)",
     "omahabeach_offensive_us": "Omaha Beach Offensive (US)",
+    "omahabeach_warfare_night": "Omaha (Night)",
```

Comment:
For consistency, this should say "Omaha Beach". Also, while "night" is the safe option here, it's technically supposed to be a "dusk" map? Same for El Alamein. Maybe something to remember for the future.